docker/image_types.yaml

```diff
@@ -0,0 +1,67 @@
+# Defines the different types of docker images that FuzzBench uses and their
+# dependency graph.
+
+'base-image':
+  tag: 'base-image'
+  context: 'docker/base-image'
+
+'base-builder':
+  tag: 'base-builder'
+  context: 'docker/base-builder'
+  depends_on:
+    - 'base-image'
+
+'base-runner':
+  tag: 'base-runner'
+  context: 'docker/base-runner'
+  depends_on:
+    - 'base-image'
```

Comment:
As suggested in #548 by @inferno-chromium , shall we change this to depend on `base-builder` here?